src/containers/MemoryGame/index.jsx

```diff
@@ -1,12 +1,44 @@
-import React from "react";
+import React, { useState } from "react";
 import "./index.css";
-import MemoryGameCards from "../../components/memorygame/MemoryGameCards/index";
+import { Col, Row } from "antd";
+import { useTranslation } from "react-i18next";
+import Confetti from "react-dom-confetti";
+import confettiConfig from "../../components/memorygame/confettiConfig";
+import MemoryGameFactList from "../../components/memorygame/MemoryGameFactList";
+import MemoryGameBoard from "../../components/memorygame/MemoryGameBoard";
 
 const MemoryGame = () => {
+  const [t] = useTranslation();
+  const [isGameCompleted, setIsGameCompleted] = useState(false);
+  // matchedCards is being used to display environmental facts.
+  // It is an array of object(states).Each object is a successfully matched card state.
+  // When a match is made only state of first card gets pushed inside matchedCards.
+  // Because matched cards have same state. Otherwise same fact would be displayed twice.
+  const [matchedCards, setMatchedCards] = useState([]);
```

Comment:
does matched here mean completed like it can no longer be flipped? I'd call this finished cards if that's the case. otherwise the word match is ambiguous

Reply:
since the whole game is about matching it is also matched cards when I try to match one card for acid rain and one card for oceans or something. that would also be considered matching

Reply:
Yes matched means it can no longer be flipped. Changes made PTAL